.github/workflows/code_coverage.yml

```diff
@@ -16,20 +16,11 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v3
         with:
-          node-version: '12.22.0'
+          node-version-file: '.nvmrc'
+          cache: 'yarn'
```

Comment:
Something is amiss here. When you check the output for the `setup-node` in [the list of jobs](https://github.com/reload/dpl-react/runs/6370168550?check_suite_focus=true) you get:

```
Extracting ...
/usr/bin/tar xz --strip 1 --warning=no-unknown-keyword -C /home/runner/work/_temp/41ea7b71-4a05-40ff-aff6-f58c5a47a92e -f /home/runner/work/_temp/6259b08f-5d88-4161-8d51-8756009b136e
Adding to the cache ...
Done
/usr/local/bin/yarn --version
1.22.18
/usr/local/bin/yarn cache dir
/home/runner/.cache/yarn/v6
Received 244 of 244 (100.0%), 0.0 MBs/sec
Cache Size: ~0 MB (244 B)
/usr/bin/tar --use-compress-program zstd -d -xf /home/runner/work/_temp/610170f2-eac7-4997-bdb3-31c13bb4df04/cache.tzst -P -C /home/runner/work/dpl-react/dpl-react
Cache restored successfully

Cache restored from key: node-cache-Linux-yarn-59206777d6cced5312c295c4995071bd7339df74a5b6149032df1fb288b3dd04
```

Note that the cache size is 0 MB. The cache is effectively empty.

The output of the post setup action is:

```
Post job cleanup.
/usr/local/bin/yarn --version
1.22.18
/usr/local/bin/yarn cache dir
/home/runner/.cache/yarn/v6
Cache hit occurred on the primary key node-cache-Linux-yarn-59206777d6cced5312c295c4995071bd7339df74a5b6149032df1fb288b3dd04, not saving cache.
````

I interpret this as the cache not being updated either.

Perhaps this will change once the PR is merged 😕 . The current cleanup is better than the situation before anyway.